NSS-Day-Cohort-42/rare-fakes-news
# Admin can approve posts

​
## Description

This changes adds a table view for admin users to view information about posts. On the page where this table is, a user can select a post or multiple posts on the table and approve them.
​
## How do I access this branch?

`git fetch --all`
`git checkout TRB-admin-posts`

install Grommet:

`npm install grommet styled-components --save`
​
## Motivation and Context


This change is requested by the product owner
​
## How Has This Been Tested?



1. logged in as Admin, click on the Post Management route at the top of the page
2. Marvel at the list before you.
3. check a post or posts whose approved status is Not Approved
4. Click Approve, see status change.
5. log out and log in as non admin user. Observe that there is no Post Management route.
​
## Types of changes

​
## Checklist:



![bitmoji](https://sdk.bitmoji.com/render/panel/a640399f-2837-485b-881e-3de373e8f801-6a89f47d-0069-419b-9704-59b7a29d8414-v1.png?transparent=1&palette=1&width=246)
